app/views/pregnancies/_call_log.html.erb

```diff
@@ -0,0 +1,16 @@
+<section id="call_log" class="call-info tab-pane active">
```

Comment:
Sorry, I'm responsible for the merge conflict here! We should go with yours -- please  resolve merge conflicts in favor of what's in here.